### PR TITLE
Updated legacy pybee references and setup docs for toga-demo

### PR DESCRIPTION
**Pull request by jess010**
Minor updates for documentation as follows:
1. Replaced legacy references to 'pybee' with 'beeware'
2. Updated the documentation for toga-demo installation to provide additional instruction on the main page for people with Mac OS Sierra+ distributions (captured in comments on issue 602).

## PR Checklist:

- [ ] All new features have been tested - N/A doc update
- [ ] All new features have been documented - N/A doc update
- [x] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

File: examples/imageview/imageview/app.py [hunk 1]
```diff
@@ -23,15 +23,15 @@ def startup(self):
         # image from remote URL
         # no style parameters - we let Pack determine how to allocate
         # the space
-        image_from_url = toga.Image('https://pybee.org/project/projects/libraries/toga/toga.png')
+        image_from_url = toga.Image('https://beeware.org/project/projects/libraries/toga/toga.png')
         imageview_from_url = toga.ImageView(image_from_url)
         box.add(imageview_from_url)
```
Comment (brutusthebee) [inline examples/imageview/imageview/app.py]: At column 1: (W293) blank line contains whitespace

File: examples/imageview/imageview/app.py [hunk 2]
```diff
@@ -23,15 +23,15 @@ def startup(self):
...
         
         self.main_window.content = box
         self.main_window.show()
 
 def main():
```
Comment (brutusthebee) [inline examples/imageview/imageview/app.py]: At column 1: (E302) expected 2 blank lines, found 1

File: examples/slider/slider/app.py [hunk 3]
```diff
@@ -62,4 +62,4 @@ def my_on_slide(self, slider):
 
 def main():
```
Comment (brutusthebee) [inline examples/slider/slider/app.py]: At column 1: (E302) expected 2 blank lines, found 1

File: examples/detailedlist/detailedlist/app.py [hunk 4]
```diff
@@ -62,7 +62,7 @@ def startup(self):
         self.main_window.show()
 
 def main():
```
Comment (brutusthebee) [inline examples/detailedlist/detailedlist/app.py]: At column 1: (E302) expected 2 blank lines, found 1

File: examples/scrollcontainer/scrollcontainer/app.py [hunk 5]
```diff
@@ -19,7 +19,7 @@ def startup(self):
         self.main_window.show()
 
 def main():
```
Comment (brutusthebee) [inline examples/scrollcontainer/scrollcontainer/app.py]: At column 1: (E302) expected 2 blank lines, found 1

File: examples/imageview/imageview/app.py [hunk 6]
```diff
@@ -1,16 +1,17 @@
 import os
```
Comment (brutusthebee) [inline examples/imageview/imageview/app.py]: At column 1: (F401) 'os' imported but unused

File: examples/scrollcontainer/scrollcontainer/app.py [hunk 7]
```diff
@@ -1,10 +1,10 @@
 import os
```
Comment (brutusthebee) [inline examples/scrollcontainer/scrollcontainer/app.py]: At column 1: (F401) 'os' imported but unused

File: examples/scrollcontainer/scrollcontainer/app.py [hunk 8]
```diff
@@ -1,10 +1,10 @@
...
 import toga
-from toga.style.pack import *
+from toga.style.pack import COLUMN, LEFT
 
 class ScrollContainerApp(toga.App):
```
Comment (brutusthebee) [inline examples/scrollcontainer/scrollcontainer/app.py]: At column 1: (E302) expected 2 blank lines, found 1

File: examples/scrollcontainer/scrollcontainer/app.py (hunk 7 above)
Comment (brutusthebee) [inline examples/scrollcontainer/scrollcontainer/app.py]: At column 1: (F401) 'os' imported but unused

File: examples/scrollcontainer/scrollcontainer/app.py (hunk 8 above)
Comment (brutusthebee) [inline examples/scrollcontainer/scrollcontainer/app.py]: At column 1: (E302) expected 2 blank lines, found 1

**Comment by jess010** [comment]
The needs of brutusthebee for this PR appear to have been satisfied - yay! Thanks for the feedback @phildini, I have incorporated this change.

**Pull request merged**